app/javascript/dashboard/routes/dashboard/helpcenter/components/PortalSettingsBasicForm.vue

```diff
@@ -16,7 +16,7 @@
             {{ $t('HELP_CENTER.PORTAL.ADD.LOGO.LABEL') }}
           </label>
           <div class="logo-container">
-            <thumbnail :username="name" size="56" variant="square" />
+            <thumbnail :username="name" size="56px" variant="square" />
```

Comment:
These changes will fix the issue -  Thumbnail UI has broken on the portal settings page.